ports/esp32/boards/LOLIN_S3/mpconfigboard.h

```diff
@@ -0,0 +1,15 @@
+#define MICROPY_HW_BOARD_NAME               "LOLIN S3"
+#define MICROPY_HW_MCU_NAME                 "ESP32S3"
+
+#define MICROPY_PY_BLUETOOTH                (0)
+#define MICROPY_PY_MACHINE_DAC              (0)
+
+// Enable UART REPL for modules that have an external USB-UART and don't use native USB.
+#define MICROPY_HW_ENABLE_UART_REPL         (1)
+
+#define MICROPY_HW_I2C0_SCL                 (41)
```

Comment:
`machine.I2C(0)` fails to initialise because of #7772:

```python
>>> import machine
>>> machine.I2C(0)
E (11156400) i2c: i2c_set_timeout(839): i2c timing value error
I2C(0, scl=41, sda=42, freq=888888)
```
I think this configuration should remain, and hopefully the problem will be resolved soon.